sources/environment/commands/command-line.dylan

```diff
@@ -8,6 +8,11 @@ Warranty:  Distributed WITHOUT WARRANTY OF ANY KIND
 
 /// Useful constants
 
+define constant $success-exit-code          = 0;
+define constant $unexpected-error-exit-code = 1;
+define constant $usage-error-exit-code      = 2;
```

Comment:
Maybe sort these by seriousness so they could be used with `IF ERRORLEVEL` https://devblogs.microsoft.com/oldnewthing/20080926-00/?p=20743

Reply:
Done